ledger/src/blockstore.rs

```diff
@@ -1904,35 +1902,33 @@ impl Blockstore {
             return true;
         };
 
-        let Some(prev_merkle_root_meta) = merkle_root_metas
```

Comment:
This is safe because `self.previous_erasure_set()` on line 1896 must have found an `ErasureMeta` for `prev_erasure_set` in order to reach here.

Reply:
Shouldn't `fn previous_erasure_set` actually return that `ErasureMeta` instead of looking it up again here?

Reply:
yeah good point, I didn't do this last time because we only needed the merkle root.
Updated it to return the meta as well.